src/ui/reducers/timeline.ts

```diff
@@ -17,47 +17,43 @@ function initialTimelineState(): TimelineState {
     stalled: false,
     timelineDimensions: { left: 1, top: 1, width: 1 },
     unprocessedRegions: [],
+    /** @deprecated This appears to be obsolete for now? */
```

Comment:
It is. Right now, startTime is always 0, endTime is always the recording duration, and scale is 1. We mostly use this as a replacement for a `duration` value now. Could use a refactor soon but just hesitant to do it/ask anybody to do it unless it's adjacent to something somebody's working on.